label_weeds.py

```diff
@@ -261,8 +326,9 @@ def run(self):
     import argparse
     parser = argparse.ArgumentParser(description='__doc__')
```

Comment:
Tisknutí `"__doc__"` nemá smysl. Tady má být `descripption`, které by bylo v `__doc__`, jenže to jsi zrušil.